### DISCUSSION TITLE
CoPilot Not Active After Making Payment

### DISCUSSION THREAD

**Discussion by retromack** [community/community#113528] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

My github copilot subscription has not been activated despite making payment for it. I have tried to contact support but i have not got any feedback 

https://support.github.com/ticket/personal/0/2685219

**Comment by Akash1134** [comment]
Hi @retromack The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by retromack** [reply]
@Akash1134 i have already opened a tickets but got no feedback for the last 24 hours already linked my ticket in the comments

**Comment by retromack** [comment]
@Akash1134 it has been a week already my issue has not been addressed via the support ticket that i created

**Reply by retromack** [reply]
https://support.github.com/ticket/personal/0/2685219

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please continue to work with support to resolve this issue.
